packages/dataviews/src/components/dataviews-bulk-actions/index.tsx

```diff
@@ -288,3 +282,19 @@ export default function BulkActions< Item >( {
 		</>
 	);
 }
+
+export default function BulkActions() {
+	const { data, actions = [], view } = useContext( DataViewsContext );
+	const hasPossibleBulkAction = useSomeItemHasAPossibleBulkAction(
+		actions,
+		data
+	);
+	if (
```

Comment:
It feels like these checks should be the consumer's responsibility, or checked by a passed prop or something..

Reply:
For the "list" view check I could agree with you but I think it's easy to revert, for now, there's a decision that list views don't support bulk actions and it's consistent across all dataviews.

For the other check, I disagree, it's something that should be bundled in the component.

Reply:
>For the other check, I disagree, it's something that should be bundled in the component.

Which one? For my preference layouts to have a prop like `supportsBulkActions`?

Reply:
> Which one? For my preference layouts to have a prop like supportsBulkActions?

Why a prop, it's something that can be guessed from the actions, the prop seems redundant to me.

Reply:
How it can be guessed by the actions?

Reply:
Like today: `useSomeItemHasAPossibleBulkAction` hook is used here to know whether to render this or not.

Reply:
Exactly, but here we don't want to render bulk actions not by the items and whether are eligible. We just don't want to render them for a specific layout.

Reply:
There's two checks here: the layout which I agree that could be a user decision If needed. But right now it's not a user decision, so this PR keeps it as is.

and the other check `useSomeItemHasAPossibleBulkAction` is not a user decision, it's bundled here as well and should always stay as is IMO.

If I'm guessing right from your comment, you think the "layout" list not showing bulk actions should be a user decision? Maybe, that's fine by me, but it's not the case in trunk and I don't want to change that here.

Reply:
Yeah, sorry for the confusion. I was only referring to `layout`. We can update later the layout checks then.